PrivacyCookie/DemoBundleBanner.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\DemoBundle\PrivacyCookie;
+
+use eZ\Publish\API\Repository\LocationService;
+use eZ\Publish\API\Repository\URLAliasService;
+use eZ\Publish\API\Repository\Values\Content\Content;
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\API\Repository\Values\Content\URLAlias;
+use EzSystems\PrivacyCookieBundle\Banner\Banner;
+use EzSystems\DemoBundle\PrivacyCookie;
+use Symfony\Component\Translation\Translator;
+
+/**
+ * Builds a Privacy Cookie Banner object based on a configured Content Object.
+ */
+class DemoBundleBanner implements DemoBundleBannerFactory
+{
+    public static function build( Content $bannerContent, LocationService $locationService, URLAliasService $urlAliasService, Translator $translator )
+    {
+        $banner = new Banner();
+
+        $banner->caption = $bannerContent->getFieldValue( 'description' )->xml->textContent;
```

Comment:
>
> > +        $banner->caption = $bannerContent->getFieldValue( 'description' )->xml->textContent;
>
> question here. why we use ->xml->textContent here? If description is a
> xmlblock and the user wants some bold text or whatever, won't will loose
> them?
>
Carlos is right. I'm not sure we should aim at supporting rich text for
caption straight away. It can be an improvement.